substrate/frame/balances/src/lib.rs

```diff
@@ -222,13 +222,13 @@ pub mod pallet {
 			type ExistentialDeposit = ConstU64<1>;
 
 			type ReserveIdentifier = ();
-			type FreezeIdentifier = ();
+			type FreezeIdentifier = Self::RuntimeFreezeReason;
```

Comment:
Seems like there is still a few pallets that don't use `TestDefaultConfig`, reasonable to update them as a part of this PR?

Reply:
> Seems like there is still a few pallets that don't use `TestDefaultConfig`, reasonable to update them as a part of this PR?

yes, good point, I will do that here

Reply:
@kianenigma I updated all (hopefully) pallets for `pallet_balances::Config and `TestDefaultConfig`, please take a look